kafka-scheduler/src/main/java/com/mesosphere/dcos/kafka/scheduler/KafkaScheduler.java

```diff
@@ -181,39 +183,36 @@ private void startApiServer() {
         });
     }
 
-    protected DefaultRecoveryScheduler createRecoveryScheduler(KafkaOfferRequirementProvider offerRequirementProvider) {
+    protected  DefaultRecoveryPlanManager createRecoveryPlanManager(KafkaOfferRequirementProvider offerRequirementProvider) {
+
         RecoveryConfiguration recoveryConfiguration = envConfig.getRecoveryConfiguration();
         LaunchConstrainer constrainer = new TimedLaunchConstrainer(
                 Duration.ofSeconds(recoveryConfiguration.getRecoveryDelaySecs()));
         RecoveryRequirementProvider recoveryRequirementProvider =
                 new KafkaRecoveryRequirementProvider(
                         offerRequirementProvider,
                         configState.getConfigStore());
-        return new DefaultRecoveryScheduler(
-                frameworkState.getStateStore(),
-                taskFailureListener,
-                recoveryRequirementProvider,
-                offerAccepter,
+
+            return new DefaultRecoveryPlanManager(frameworkState.getStateStore(), recoveryRequirementProvider,
```

Comment:
- indentation of `return new ...` line is off
- could continue to have all params on separate lines like before